orm/internal/testpb/bank.proto

```diff
@@ -9,30 +9,30 @@ import "cosmos/orm/v1/orm.proto";
 message Balance {
   option (cosmos.orm.v1.table) = {
     id: 1;
-    primary_key: {
-      fields:
-          "address,denom"
-    }
-    index: {
-      id:
-          1 fields: "denom"
-    }
-  };
+primary_key: {
+fields:
```

Comment:
Same comment as here: https://github.com/cosmos/cosmos-sdk/pull/12643#issuecomment-1189654608

I find it actually looks worse. Is there a reason it format it like that?

Reply:
Not that I know, and in terms of the cosmetics, I agree. 

Maybe it is buf?

to make this pr, I did only:

`make proto-gen`